docs/Concepts/Mining.md

```diff
@@ -0,0 +1,6 @@
+description: Mining overview
+<!--- END of page meta data -->
+
+Hyperledger Besu supports CPU and GPU mining which is [configured using command line options](../HowTo/Configure/Configure-Mining.md).
+
+Besu supports GPU mining using [Ethminer](https://github.com/ethereum-mining/ethminer) with the `stratum+tcp` or `http` (getwork mode) schemes.
```

Comment:
I would prefer to re-word this as "has been tested with" than make a "supports" claim.  I've made other software work (claymore to be specific).

Reply:
Fixed.